### DISCUSSION TITLE
Can there be a stable-3_4_0 branch?

### DISCUSSION THREAD

**Discussion by SydneyUni-Jim** [pkp/plagiarism#60] [Q&A] [answered]
Can there be a `stable-3_4_0` branch that points to the latest released version of the plugin for OJS 3.4?

Other plugins — such as [quickSubmit](https://github.com/pkp/quickSubmit.git) and [pln](https://github.com/pkp/pln.git) — have this branch. And this plugin does have a `stable-3_3_0` branch.

I had been relying on these branches to checkout the latest versions of plugins when rebuilding my OJS container image, which has these plugins baked into it.

**Comment by touhidurabir** [comment, marked as answer]
Yes , there will be a stable-3.4.0 branch once we merge the PR associated with issue https://github.com/pkp/plagiarism/issues/52 .
